installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -8,77 +8,79 @@ resource "null_resource" "chef_provision_jenkins_server" {
     type = "ssh"
     private_key = "${file("${lookup(var.jenkinsservermap, "jenkins_ssh_key")}")}"
   }
-  
-  #BEGIN chef cookbook edits TODO consider moving these to their own .tf file
-  #Because we have to provision a preexisting machine here and can't use the terraform ses command,
-  #we must use sed to insert AWS creds from the provisioner environment into a script chef will run later, before we copy the cookbook to the remote box.
+
   provisioner "local-exec" {
-    command = "sed -i 's/AWS_ACCESS_KEY=.*.$/AWS_ACCESS_KEY='${var.aws_access_key}'/g' ${var.cookbooksSourceDir}/jenkins/files/credentials/aws.sh"
+    command = "${var.configureJazzCore_cmd} ${var.envPrefix} ${var.cognito_pool_username}"
```

Comment:
@suryajak configureJazzCore_cmd has been removed as part of code reorganization #255. @bleggett Please update.